pyproject.toml

```diff
@@ -71,3 +72,19 @@ build-backend = "poetry.core.masonry.api"
 [tool.coverage.run]
 branch = true
 source = ["xrpl"]
+
+[tool.coverage.report]
+show_missing = true
```

Comment:
I recommend adding `skip_empty = true` and `skip-covered = true` to make the report a little easier to read.